### PR TITLE
修正了多色模式下，背景色透明导致的内容穿透

### PR DESCRIPTION
**Pull request by Carlosfengv**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [ ] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes #

### Changelog
🇨🇳 Chinese
- 修复了 Notification 多色模式下，背景色透明导致的内容穿透

---

🇺🇸 English
- Fix ...

### Checklist
- [ ] Test or no need
- [ ] Document or no need
- [ ] Changelog or no need

### CODE REVIEW

File: packages/semi-foundation/notification/variables.scss [hunk 1]
```diff
@@ -15,6 +15,7 @@ $color-notification_info_light-bg: var(--semi-color-info-light-default); // 彩
 $color-notification_info_light-border: var(--semi-color-info); // 彩色通知信息描边色
 $color-notification_danger_light-bg: var(--semi-color-danger-light-default); // 彩色通知危险背景色
 $color-notification_danger_light-border: var(--semi-color-danger); // 彩色通知危险描边色
+$color-notification-bg-ambient: var(--semi-color-bg-0); // 透明背景色叠加层(与bg0保持一致不建议修改)
```
Comment (pointhalo) [inline packages/semi-foundation/notification/variables.scss:18]: @shijiatongxue 确认一下这个命名。按照scss变量命名规则，是不是应该把bg放后面？

Comment (shijiatongxue) [reply packages/semi-foundation/notification/variables.scss:18]: 这个 ambient 啥意思 @Carlosfengv

File: packages/semi-foundation/notification/notification.scss [hunk 2]
```diff
@@ -93,23 +93,23 @@ $module: #{$prefix}-notification;
 
         &-light {
             &.#{$module}-notice-warning {
-                background-color: $color-notification_warning_light-bg;
+                background: linear-gradient(0deg, $color-notification_warning_light-bg, $color-notification_warning_light-bg), $color-notification-bg-ambient;
```
Comment (shijiatongxue) [inline packages/semi-foundation/notification/notification.scss:96]: 最后一个为啥在括号外面，没看懂这个语法。

File: packages/semi-foundation/notification/variables.scss (hunk 1 above)
Comment (Carlosfengv) [reply packages/semi-foundation/notification/variables.scss:18]: 叠加的背景色，主要为了引入 --color-bg-0 的全局变量

Comment (shijiatongxue) [reply packages/semi-foundation/notification/variables.scss:18]: 这里改为 color-notification_ambient-bg ？

Comment (Carlosfengv) [reply packages/semi-foundation/notification/variables.scss]: 已改

**Pull request merged**